### ISSUE TITLE
Hetzner is not connected with Robotoff anymore

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-infrastructure#30]
Hetzner (hosting the image models) is not connected with Robotoff anymore

**Comment by CharlesNepote** [comment]
1. - [x] Create new machine to host it: "ai-off" (10 cores and 64 GB RAM).
2. - [x] give access to "ai-off" to @raphael0202
just `ssh -i ~/.ssh/id_rsa -J raphael0202@ovh1.openfoodfacts.org raphael0202@10.1.0.103`
3. - [ ] Optionally, document deployment or migration
4. - [ ] Migrate or deploy.

I have done the two first steps. I can do the rest if it's documented anywhere (and if I have access to Hetzner machine).

**Comment by ocervell** [comment]
@teolemon @kulizhsy maybe for an update

**Comment by kulizhsy** [comment]
Hetzner up to 2 weeks ago was hosting the Tensorflow Serving and the ANN service for image clustering.

Both of these have been migrated to `ovh1.openfoodfacts.org`, to the `robotoff` VM.
An additional replica was also added to `robotoff-dev`, for the pre-prod testing.

As far as I am aware we shouldn't need `ai-off` for Robotoff, nor have I done any changes on that VM (Robotoff was configured to talk to Hetzner when I started the migration).
